app/views/includes/_head.html.erb

```diff
@@ -0,0 +1,6 @@
+<title>ShinySite</title>
```

Comment:
Consider using the `@page_title` variable in the `<title>` element of `_head.html.erb` templates when its available.